### PR TITLE
Rename AABB `get_area` to `get_volume`

### LINKED ISSUE #31692: Why the get_area() function of AABB class is named get_area, instead of being get_volume()?
The shows that it is return the volume of the AABB, yet the function is named get_area(). Can anyone tell me why it named that.

https://github.com/godotengine/godot/blob/2f63811c9aa1d1710dc045e01b1391493243ddde/core/math/aabb.cpp#L35

### PR DESCRIPTION
**Pull request by briansemrau**
Fixes #31692

Should be added to #16863?

### CODE REVIEW

**Review by AaronRecord — APPROVED** [review]

File: servers/physics_3d/godot_shape_3d.h [hunk 1]
```diff
@@ -256,7 +256,7 @@ class GodotCylinderShape3D : public GodotShape3D {
 	_FORCE_INLINE_ real_t get_height() const { return height; }
 	_FORCE_INLINE_ real_t get_radius() const { return radius; }
 
-	virtual real_t get_area() const override { return 4.0 / 3.0 * Math_PI * radius * radius * radius + height * Math_PI * radius * radius; }
+	virtual real_t get_volume() const override { return 4.0 / 3.0 * Math_PI * radius * radius * radius + height * Math_PI * radius * radius; }
```
Comment (briansemrau) [inline servers/physics_3d/godot_shape_3d.h:259]: Not relevant to the goals of this PR, but this calculation looks wrong.
```suggestion
	virtual real_t get_volume() const override { height * Math_PI * radius * radius; }
```

Comment (briansemrau) [reply servers/physics_3d/godot_shape_3d.h:259]: Moved to a separate issue: #54636

**Review by akien-mga — APPROVED** [review]
Seems good to merge, once rebased.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**